Security/src/EOMT.ps1

```diff
@@ -790,7 +790,8 @@ try {
         #Normal run
         $IsVulnerable = Get-ServerVulnStatus
         if ($IsVulnerable) {
-            $Message = "$env:computername is vulnerable: applying mitigation"
+            $Message = "$env:computername is vulnerable: needs to apply mitigation"
+            $Message + "`n" + "Please use Microsoft Update to install the latest Security Update For Exchange Server (KB5000871)"
```

Comment:
This change in wording is incorrect: we are applying the mitigation with script; the user needs to install the patch for long-term support.

I also worry that if we just say "please use MU to install the update" users will be confused as to whether this script actually provided any protection, so we should be clear in the message that we *have* applied the mitigation.

You can use something like "EOMT will attempt to mitigate against known attacks. For long-term protection, please use Microsoft Update to install the latest Security Update for Exchange Server (KB5000871)."